src/poliastro/threebody/cr3bp_char_quant.py

```diff
@@ -0,0 +1,105 @@
+"""
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+"""
+from poliastro.core.threebody.cr3bp_quantities_calculations import (
+    calculate_mu,
+    calculate_tstar,
+)
+
+
+class SystemChars:
+    """Computes and stores the properties (mu, l*, t*) of a CR3BP system
+    'mu': mass ratio of P1-P2 primary bodies
+    'l*': characterisitic lenght of P1-P2 system
+    't*': characterisitic time of P1-P2 system
+    If P2 is more massive than P1 then swap the Pi, so that P1 is the more massive body
+    """
+
+    def __init__(self, name, mu, lstar, tstar):
+        """
+        Constructor
+
+        Parameters
+        ----------
+        name: string
+            System name, format: 'Body1Body2', e.g. 'EarthMoon', 'SunEarth'
+        mu: float, dimensionless
+           mass ratio of P1-P2 primary bodies
+        lstar: float, km
+           Characterisitc length of P1-P2 system
+        tstar: float, sec
+            Characterisitc time of P1-P2 system
+        """
+        self.name = name
+        self._mu = mu
+        self._lstar = lstar
+        self._tstar = tstar
+
+    @classmethod
+    def from_primaries(cls, p1, p2):
+        """
+        Computes and sets the characteristic quanitites based on p1 and p2 bodies
+        Parameters
+        ----------
+        p1: ~poliastro.bodies.Body
+        p2: ~poliastro.bodies.Body
+        """
+
+        name, mu, lstar, tstar = cls.bodies_char_compute(p1, p2)
+        return cls(name, mu, lstar, tstar)
+
+    @property
+    def bodies_char_compute(self, p1, p2):
+        """
+        Calculates mu, lstar, and tstar of the 'p1' and 'p2' system
+        Also, if M2>M1, then swaps p1 and p2, so that M1>M2
```

Comment:
```suggestion
        Calculates mu, lstar, and tstar of the 'p1' and 'p2' system

        Also, if M2>M1, then swaps p1 and p2, so that M1>M2
```